safety_gymnasium/wrappers/__init__.py

```diff
@@ -16,4 +16,17 @@
 
 from safety_gymnasium.wrappers.autoreset import SafeAutoResetWrapper
 from safety_gymnasium.wrappers.env_checker import SafePassiveEnvChecker
+from safety_gymnasium.wrappers.gymnasium_conversation import (
```

Comment:
```suggestion
from safety_gymnasium.wrappers.gymnasium_conversion import (
```
will `conversion` be better?

Reply:
Yep. Conversation and conversion are two different words with distinct meanings:

Conversation refers to a verbal or written exchange of ideas, thoughts, or information between two or more people. It involves an informal or formal discussion or dialogue where people share their views and opinions.

Conversion, on the other hand, refers to the act of changing something from one form or state to another. It can refer to a change in beliefs, attitudes, behavior, or even the physical characteristics of something.

@XuehaiPan @muchvo. I think conversion will be more reasonable.